packages/core/pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "rpaframework-core"
-version = "6.4.0"
+version = "6.5.0"
```

Comment:
I tested locally with a wheel, but in this PR, should I split this change in another PR? (along with all the other `core` changes)

Reply:
We decided internally that's best to put all changes in one PR and then release updated dependencies to PyPI once the relevant CI for them passes here. (and once their changes are approved and stay backwards compatible)